hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -82,15 +86,27 @@ public void configure() {
                 .handled(true)
                 .id("ValidationException");
         
+        onCompletion().choice()
+        .when(header("isFileDropsAllowed")
+        		.isEqualTo("Y"))
+        .bean(V2FileDrops.class).id("V2FileDrops");
```

Comment:
Some small issues with the merge I think. This class doesn't appear to be committed/exist